locales/en.yml

```diff
@@ -57,37 +197,28 @@ city:
   change: Change city
 role:
   label: Role
-profile:
```

Comment:
`profile` was already here. Just extend fields here.
And whenever you remove existing translations ensure they were not used in other places.

Reply:
Corret.